### PR TITLE
Get rename metrics for non nested scenario for hns bucket

### PR DESCRIPTION
**Pull request by anushka567**
### Description
Added _run_rename_function which gets rename metrics for non nested scenario for hns bucket as well as flat bucket.

### Link to the issue in case of a bug fix.
NA

### Testing details
1. Manual - NA
2. Unit tests - [here](https://github.com/GoogleCloudPlatform/gcsfuse/blob/288a014826ee02c0a94a34ea9c2d3c75c1a8a9dd/perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark_test.py#L184)
3. Integration tests - NA

### CODE REVIEW

File: perfmetrics/scripts/hns_rename_folders_metrics/config-flat.json [hunk 1]
```diff
@@ -0,0 +1,93 @@
+{
+  "name": "hns-rename-benchmark-flat" ,
```
Comment (Tulsishah) [inline perfmetrics/scripts/hns_rename_folders_metrics/config-flat.json:2]: a

File: perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py [hunk 2]
```diff
@@ -275,7 +277,20 @@ def _perform_testing(dir, test_type, num_samples, results):
...
   if test_type == "hns":
-    # TODO add mount function for test type hns
+    # Creating config file for mounting with hns enabled.
+    with open("config.yml",'w') as mount_config:
+      mount_config.write("enable-hns: true")
+    hns_mount_flags="--config-file=config.yml"
+    hns_bucket_name=mount_gcs_bucket(dir["name"], hns_mount_flags, log)
+
+    # Record time of operation and populate the results dict.
+    flat_results = _record_time_of_operation(hns_bucket_name, dir, num_samples)
```
Comment (Tulsishah) [inline perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py:287]: nit: hns_result

File: perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py [hunk 3]
```diff
@@ -339,20 +352,48 @@ def _parse_arguments(argv):
...
-    print(upload_values_flat)
+    print(upload_values)
+
+
+if __name__ == '__main__':
+  argv = sys.argv
+  if len(argv) < 2:
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 renaming_benchmark.py  [--upload_gs] [--num_samples NUM_SAMPLES] config_file ')
```
Comment (Tulsishah) [inline perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py:383]: Let's pass `--bucket-type` as an argument and pass only one config-file at a time. This way, we can avoid the "if" conditions here.

Comment (anushka567) [reply perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py]: Updated.

File: perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py [hunk 4]
```diff
@@ -339,20 +351,39 @@ def _parse_arguments(argv):
...
     if exit_code != 0:
       log.error("Upload to gsheet failed!")
   else:
-    print(upload_values_flat)
+    print(upload_values)
+
+
+if __name__ == '__main__':
+  argv = sys.argv
+  if len(argv) < 2:
```
Comment (Tulsishah) [inline perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py:379]: It should be 5

File: perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py [hunk 5]
```diff
@@ -339,20 +351,39 @@ def _parse_arguments(argv):
...
+    raise TypeError('Incorrect number of arguments.\n'
+                    'Usage: '
+                    'python3 renaming_benchmark.py  [--upload_gs] [--num_samples NUM_SAMPLES] config_file bucket_type')
+
+  args = _parse_arguments(argv)
+  #check_dependencies(['gcloud', 'gcsfuse'], log)
```
Comment (Tulsishah) [inline perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py:385]: Unblock this

File: perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py (hunk 4 above)
Comment (anushka567) [reply perfmetrics/scripts/hns_rename_folders_metrics/renaming_benchmark.py]: Updated to 3, since upload_gs and num_samples flags are optional.

**Review by Tulsishah — DISMISSED** [review]

File: .github/header-checker-lint.yml [hunk 6]
```diff
@@ -23,7 +23,6 @@ allowedLicenses:
 sourceFileExtensions:
   - 'go'
   - 'Makefile'
-  - 'json'
```
[the remark this answers is not in the document]
Comment (anushka567) [reply .github/header-checker-lint.yml:26]: to exempt it from header checks,.. json does not support comments so cannot add license header.

**Review by ankitaluthra1 — APPROVED** [review]

**Pull request merged**